ruby_programming/basic_ruby/debugging.md

```diff
@@ -113,16 +116,17 @@ To follow along with these examples save the code into a Ruby file (e.g., `scrip
 ~~~ruby
 require 'pry-byebug'
 
-def double_words_in_phrase(string)
-  string_array = string.split(' ')
+def isogram?(string)
+  original_length = string.length
+  string_array = string.downcase.split
 
   binding.pry
 
-  string_array.map { |word| word * 2 }
-  string_array.join(' ')
+  unique_length = string_array.uniq.length
+  original_length == unique_length
 end
 
-double_words_in_phrase("This is a test")
+isogram?("Odin")
 ~~~
 
 When your code executes and gets to `binding.pry`, it will open an IRB-like session in your terminal. You can then use that session to check the values of anything within the scope of where you included `binding.pry`. However, keep in mind that any code written *after* the `binding.pry` statement will not have been evaluated during the Pry session. Thus, adding a `binding.pry` line in our code is similar to creating a breakpoint in JavaScript.
```

Comment:
"anything within the scope"
I know this was part of the original lesson, but I think this wording is misleading. What do you think of clarifying this and maybe adding specific examples of what values will be available (`original_length` and `string_array`, but not `unique_length`)? I think this paragraph would be a bit more meaningful with examples.

Reply:
Yeah. It'll be useful. I'll add it.

Reply:
When your code executes and gets to `binding.pry`, it will open an IRB-like session in your terminal. You can then use that session to check the values of anything within the scope of where you included `binding.pry`. However, keep in mind that any code written *after* the `binding.pry` statement will not have been evaluated during the Pry session. 

For example, here `original_length` and `string_array` are in scope but not `unique_length` that is written after `binding.pry` and it's not evaluated yet. 

Thus, adding a `binding.pry` line in our code is similar to creating a breakpoint in JavaScript.


**Is this satisfies our need?**

Reply:
You are quick! I'd like to break up that long sentence into two, because it is slightly hard to follow. Maybe something like:

...are in scope. However, `unique_length` is not in scope, because it is written after `binding.pry` and has not been evaluated yet.

Reply:
Done. I've updated it.